lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js

```diff
@@ -123,4 +124,64 @@
         let o = __chakraLibrary.Object(this);
         return __chakraLibrary.CreateArrayIterator(o, 2 /* ArrayIterationKind.KeyAndValue*/);
     });
-});
\ No newline at end of file
+
+    platform.registerFunction(FunctionsEnum.ArrayIndexOf, function (searchElement, fromIndex) {
+        // ECMAScript 2017 #sec-array.prototype.indexof
+        "use strict";
+
+        if (this === null || this === undefined) {
```

Comment:
Isn't the 'prototype.includes' is similar as indexOf? may be in next iteration. This looks solid.